internal/pkg/levee/levee_test.go

```diff
@@ -34,3 +34,14 @@ func TestLevee(t *testing.T) {
 	}
 	analysistest.Run(t, dataDir, Analyzer, "./...")
 }
+
+func TestLeveeDoesNotCreateReportsForPanicIfAllowPanicOnTaintedValuesIsSet(t *testing.T) {
```

Comment:
`IsSet` seems imprecise to me. Suggestion: `...IfPanickingOnTaintedValuesIsAllowed`.

Reply:
Done.